src/main/java/org/assertj/core/internal/Paths.java

```diff
@@ -496,4 +499,22 @@ private static void assertExpectedEndPathIsNotNull(final Path end) {
     requireNonNull(end, "the expected end path should not be null");
   }
 
+  public void assertIsEmptyPath(AssertionInfo info, Path actual) {
```

Comment:
To avoid confusion with directories, I suggest to rename this one:
```suggestion
  public void assertIsEmptyFile(AssertionInfo info, Path actual) {
```